pkg/rego/metadata.go

```diff
@@ -227,9 +231,44 @@ func (m *MetadataRetriever) updateMetadata(meta map[string]interface{}, metadata
 			}
 		}
 	}
+
+	var err error
+	if metadata.CloudFormation, err = m.getEngineMetadata("cloud_formation", meta); err != nil {
+		return err
+	}
+
+	if metadata.Terraform, err = m.getEngineMetadata("terraform", meta); err != nil {
+		return err
+	}
+
 	return nil
 }
 
+func (m *MetadataRetriever) getEngineMetadata(schema string, meta map[string]interface{}) (*scan.EngineMetadata, error) {
+	var sMap map[string]interface{}
+	if raw, ok := meta[schema]; ok {
+		sMap, ok = raw.(map[string]interface{})
+		if !ok {
+			return nil, fmt.Errorf("failed to parse %s metadata: not an object", schema)
+		}
+	}
+
+	var em scan.EngineMetadata
+	if sMap["good_examples"] != nil {
+		em.GoodExamples = []string{fmt.Sprintf("%s", sMap["good_examples"])}
```

Comment:
maybe its good add a cast check , for example : 
```
val, ok := sMap["good_examples"].(string)
	if ok {
             em.GoodExamples = []string{fmt.Sprintf("%s",val)}
         }
```
otherwise this could happen incase of  `sMap["good_examples"] = 1`

`em.GoodExamples = []string{%!s(int=1)}`

Reply:
thanks, addressed.